media-gfx/converseen/files/converseen-0.9.11.1-graphicsmagick-support.patch

```diff
@@ -0,0 +1,51 @@
+--- a/CMakeLists.txt
```

Comment:
Put in a bug reference to the Gentoo bug at the top. I know you're going to polish this more later for upstreaming, but still. Thanks!

Reply:
Thanks for the review @thesamesam !
I added the bug reference.